src/mixins/index.js

```diff
@@ -1,13 +1,10 @@
 import ClickAwayable from './click-awayable';
-import StylePropable from './style-propable';
```

Comment:
this is breaking change. I think we shouldn't deprecate the export. but rather the usage. you can put the warnings in merge and prepare functions. so you wouldn't have to remove these to avoid warnings.